.github/CODEOWNERS

```diff
@@ -107,7 +111,7 @@ assets/               @DataDog/agent-integrations
 
 # Database monitoring
 **/base/utils/db/utils.py                           @DataDog/agent-integrations @DataDog/database-monitoring
-datadog_checks_base/tests/**/test_db_util.py        @DataDog/agent-integrations @DataDog/database-monitoring
+datadog_checks_base/tests/**/test_util.py           @DataDog/agent-integrations @DataDog/database-monitoring
```

Comment:
Noticed this one was the wrong filename so I fixed that too in line with fixing ownership of the dbm-owned base files.